schema.json

```diff
@@ -173,8 +176,23 @@
               }
             }
           },
-          "requirements": {
-            "description": "Specific requirements for this question",
+          "required": {
+            "description": "Whether the task is required to be done",
```

Comment:
What does this mean? This sentence reads as if it would expect a boolean value.

Reply:
The rationale behind this change was to split 'requirements' up into 

- 'depends_on': This specifies tasks that must be completed before executing the current task. 

- 'required': This indicates the necessity of the task itself. This could indeed be represented as a boolean (true/false) or categorized (e.g., 'mandatory', 'conditional').

Reply:
I see, but shouldn't the schema then capture that? (it's now an array of objects with text/urn fields)

Reply:
Should this become an enum?

Reply:
Both options have their pros and cons. Alternatively, we can keep it simple and efficient by making it a boolean, though this will reduce flexibility and specificity.